### PR TITLE
Convert prints to logging using --verbose flag

### PR DESCRIPTION
**Pull request by matthew-schultz**
related issue #74

- converted print() statements to logging.info()
- added an argument flag '--verbose' or '-v' to  'api' and 'extract' scripts that enables those statements to be displayed in the console and written to error.log file
   - ERROR messages are printed to screen and written to error.log by default
   - usage example: ```python3 api_egauge.py --verbose```
- error.log now rotates to a new file when it reaches 100 MB
    - for each api/extract script up to 5 old error.log files are stored + current one to limit storage usage to <=600 MB/script

### CODE REVIEW

**Comment by carlosparadis** [comment]
@matthew-schultz Where does the error log file currently resides? Was this added to the wiki?

**Comment by matthew-schultz** [comment]
@carlosparadis 

The error log file(s) are located in their associated <script_folder>/script directories
(e.g. a project's egauge error log would be at <project-name>/egauge/script/error.log)

Yeah, the error log file location is listed in the Troubleshooting section of the wiki under case __2.One of my existing sensors in sensor_info is running but not acquiring data__ https://github.com/erdl/lonoa/wiki/Lonoa-User-Guide

File: webctrl/script/api_webctrl.py [hunk 1]
```diff
@@ -10,15 +10,41 @@
...
+logging.basicConfig(level=log_level, format=__file__+': %(message)s')
+
+# Create a handler that writes log messages to error.log file
+# rotates error.log every time it reaches 100 MB to limit space usage; keeps up to 5 old error.log files
+rotating_file_handler = logging.handlers.RotatingFileHandler('error.log', maxBytes=100000000, backupCount=5)
+# set the message and date format of file handler
+formatter = logging.Formatter(fmt='%(asctime)s %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p')
+rotating_file_handler.setFormatter(formatter)
+# add the handler to the root logger
+logging.getLogger('').addHandler(rotating_file_handler)
```
Comment (carlosparadis) [inline webctrl/script/api_webctrl.py]: This needs to become a function. We should also discuss a bit later tonight on the overall code modules, now that we are more or less at this point.

**Comment by matthew-schultz** [comment]
@carlosparadis Just pushed the updates converting the logging code block to a function.

**Pull request merged**